packages/wdio-cli/src/utils.ts

```diff
@@ -287,6 +294,17 @@ export function getCapabilities(arg: ReplCommandArguments) {
         return { capabilities: { browserName: 'Chrome', ...ANDROID_CONFIG, ...optionalCapabilites } }
     } else if (/ios/.test(arg.option)) {
         return { capabilities: { browserName: 'Safari', ...IOS_CONFIG, ...optionalCapabilites } }
+    } else if (/(js|ts)$/.test(arg.option)) {
+        const config = new ConfigParser()
+        config.autoCompile()
+        config.addConfigFile(arg.option)
+        let requiredCaps = config.getCapabilities()
+        if (Object.prototype.hasOwnProperty.call(arg, 'capabilities')) {
```

Comment:
better:
```suggestion
        if (typeof arg.capabilities !== 'undefined') {
```
because it is more readable imo